### PR TITLE
User follow

### PR DESCRIPTION
**Pull request by NaokiIwamoto**
## Description / Purpose (checkpoints)
User can follow other user.

Fixes issue: 
<link-to-your-spreadsheet> 
https://docs.google.com/spreadsheets/d/1KanXdYJuprUERp6mQZAxVSb1ssrIL67G0Akeesurc4E/edit?usp=sharing
<task-number>
8

## How To Test This PR
 1. Go "Home page".
 2. Click "Users" button.
 3. Click "follow" button.
 
 **Commands**
・php artisan migrate
・create user 2 or more.

 **Screenshots**
<Home page>
[Screenshot from Gyazo](https://gyazo.com/df092a8b7088829dd2cf6a45d752fddb)

<Users page>
[Screenshot from Gyazo](https://gyazo.com/96a458a45deabee711b964b1658098ca)

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -51,9 +51,12 @@
 
 Route::get('/user/{id}/follow', 'UserController@follow')->name('follow');
 Route::get('/user/{id}/unfollow', 'UserController@unfollow')->name('unfollow');
+<<<<<<< Updated upstream
```
Comment (AubreyV) [inline routes/web.php]: Please do not push conflicts.

File: routes/web.php [hunk 2]
```diff
@@ -51,9 +51,12 @@
...
 
 Route::get('/user/lesson/category={category_id}&difficulty={difficulty}', 'LessonController@lesson_create')->name('lesson_create');
 Route::get('/user/lesson/show/lesson={lesson_id}&index={index}', 'LessonController@lesson_show')->name('lesson_show');
 Route::get('/user/lesson/answer/lesson={lesson_id}&index={index}&option={option_id}', 'LessonController@lesson_answer')->name('lesson_answer');
 Route::get('/user/lesson/result/lesson={lesson_id}', 'LessonController@lesson_result')->name('lesson_result');
 Route::get('/user/learned/{id}', 'LessonController@learned_words')->name('learned_words');
+=======
```
Comment (AubreyV) [inline routes/web.php]: Please do not push conflicts.

File: routes/web.php [hunk 3]
```diff
@@ -51,9 +51,12 @@
...
+>>>>>>> Stashed changes
```
Comment (AubreyV) [inline routes/web.php]: Please do not push conflicts.

**Review by hayato0309 — APPROVED** [review]
Approved PR#8
1. User can follow/unfollow the other users.

**Pull request merged**